tests/shared.py

```diff
@@ -52,25 +58,48 @@ def __init__(self, n):
 
         self.mesh = pybamm.Mesh(self.geometry, self.submesh_types, self.submesh_pts)
 
+        self.spatial_methods = {"negative particle": pybamm.FiniteVolume}
```

Comment:
```suggestion
        self.spatial_methods = {"negative particle": pybamm.SpatialMethodForTesting}
```

Reply:
Making this small change will break the spherical tests later. This will be dealt with properly in #153 so leaving for now.